mutters-core/src/main/java/com/rabidgremlin/mutters/core/NoIntentMatch.java

```diff
@@ -0,0 +1,33 @@
+/* Licensed under Apache-2.0 */
+package com.rabidgremlin.mutters.core;
+
+import java.util.HashMap;
+
+/**
+ * An IntentMatch used to indicate no good match was found by an IntentMatcher.
+ * 
+ * @author rabidgremlin
+ *
+ */
+public class NoIntentMatch extends IntentMatch
```

Comment:
so to test for ```NoIntentMatch```
should I use ```instanceof NoIntentMatch``` OR ```getIntent == Intent.NONE``` ??

Reply:
maybe ```IntentMatch``` needs a method ```boolean matched()``` ??

Reply:
maybe add a ```boolean didMatch()``` method

Reply:
```getIntent == Intent.NONE``` looks cleaner and as long as the IntentMatchers always return Intent.NONE shouldn't be any issue.

Reply:
Where would ```matched()``` method be used ? Just to wrap up ```getIntent == Intent.NONE``` check ?

Reply:
yeah

Reply:
Implemented.